components/contact/faq.tsx

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+
+import { useState } from 'react';
+import { FaqProps } from '@interfaces/recruitment/Faq';
+import Accordion from '@components/contact/accordion';
+
+type TabName = 'General' | 'Blockchain' | 'Machine Learning' | 'Software Development' | 'Operations';
+
+const FAQ = ({ departmentFaq }: FaqProps) => {
+  const [activeTab, setActiveTab] = useState<TabName>('General');
+
+  const handleTabClick = (tabName: TabName) => {
+    setActiveTab(tabName);
+  };
+
+  const filteredDepartment = departmentFaq.find(
+    (department) => department.departmentName === activeTab
+  );
+
+  return (
+    <div className="mt-8">
+      <div className="mx-auto grid max-w-screen-xl grid-cols-10 gap-4">
+        {/* Left column */}
+        <div className="col-span-3 rounded-lg bg-[#E6F0F0] p-8 hidden md:block">
+          <div className="flex h-full flex-col justify-start gap-12">
+            <div
+              className={`tab ${activeTab === 'General' ? 'active' : ''}`}
+              onClick={() => handleTabClick('General')}
+            >
+              <span className="tab-label">
+                {activeTab === 'General' ? (
+                  <strong className="text-3xl text-[#036167]">General</strong>
+                ) : (
+                  <span className="text-3xl text-[#004F4E]">General</span>
+                )}
+              </span>
+            </div>
```

Comment:
Could we either make the category font size or column width responsive? Currently, the words are broken up for 768px - 1125px (Software Development strong).
<img width="574" alt="Screenshot 2023-07-12 at 1 52 28 PM" src="https://github.com/NUS-Fintech-Society/SD_Fintech_Website_Revamp/assets/8915196/f00205a7-9767-4ffa-b73b-8d92a46085fb">

Reply:
Hi! Sorry for the delay! Changes have been made.